config/styleguide/config.js

```diff
@@ -0,0 +1,61 @@
+/*! Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license. */
+const path = require('path');
+const docgen = require('react-docgen');
+const docgenParser = require('react-docgen-typescript');
+const webpackConfig = require('./webpack.config.js');
+
+const getComponentPathLine = (componentPath) => {
+  const ext = path.extname(componentPath);
+  const name = path.basename(componentPath, ext);
+  const dir = path.dirname(componentPath).replace(/^src\//, 'yamui/dist/');
+  return `import ${name} from '${dir}';`;
+};
+
+const root = path.resolve(__dirname, '../../');
+
+module.exports = {
+  title: 'YamUI',
+  sections: [
+    {
```

Comment:
might be a good idea to create new sections at the top for overview documentation and usage guidelines, maybe we can pull some of that out of the project README and just point to github.io.

we might also consider sections for types of components - layout, interactive, messages... i think @annr had also mentioned interest in this early on when we were limited by storybook.

we can continue improving this after we merge.